presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -650,6 +650,30 @@ public void testArrayJoin()
         assertInvalidFunction("ARRAY_JOIN(ARRAY [cast(row(1, 2) AS row(col0 bigint, col1 bigint))], '-')", INVALID_FUNCTION_ARGUMENT);
     }
 
+    @Test
+    public void testArraySum()
+    {
+        assertFunction("ARRAY_SUM(ARRAY [1, 2, 3])",INTEGER,6);
```

Comment:
Each of these should be a separate method that can pass or fail independently.

Yes, I know presto does this wrong in many existing tests. No reason to compound the problem.